modules/contributor-guide/partials/proc_adding-a-che-theia-plug-in-into-a-workspace.adoc

```diff
@@ -7,67 +7,64 @@
 
 You can add Che-Theia plug-in into the Che-Theia IDE in three ways:
 
-* Using the Che-Theia dashboard
-* By configuring a workspace
-* At runtime
+* At Theia runtime
+* By configuring a workspace devfile
+* By preparing shareable Devfile
 
+[id="adding-a-plug-in-at-runtime_{context}"]
+== Adding a plug-in at Theia runtime
 
-[id="adding-a-plug-in-through-the-dashboard_{context}"]
-== Adding a plug-in through the dashboard
+This method adds a plug-in to an already running workspace. The plug-in is not added permanently; it is removed when the workspace stops. This is a good method to test a plug-in without affecting the permanent workspace configuration.
 
-. Navigate to the *Workspaces* tab in the dashboard.
-. Select the workspace to which you want to add the plug-in.
-. In *Workspace > _<workspace-name>_* window, click the *Plugins* tab, and select the required plug-ins.
+. Run the `Deploy Plugin by Id` command, and specify the ID of the plug-in that you want to add to the workspace. A plug-in ID can be:
```

Comment:
```suggestion
. Run the `Deploy Plugin by Id` command, and specify the ID of the plug-in that to add to the workspace. A plug-in ID can be:
```

Reply:
This method actually downloads and deploys a VS Code extension, not a plugin (in term of Che).
So, this section should be rephrased somehow. At least all the `plugin` should be replaced on `VS Code extension`

Reply:
There is some mess with namings in Che-Theia.  Command `Deploy Plugin by Id` deploys VS Code extension, but for Che-Theia it's sounds like a plugin.

Reply:
Applied

Reply:
We agreed with Vitaliy that it makes more sense to create a separate issue and address confusion between plugin and VSCode Extension